src/components/Order/Order.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+const Order = (props) => { 
+  const { order } = props;
+
```

Comment:
Here you can destructure all the information that you need at the top and just call the values bellow, also you avoid to break the demeter law:

```
const { details: { card, name } } = order
const { card: { country, email } } = details
```